GETTING_STARTED.md

```diff
@@ -64,7 +64,7 @@ to understand their behavior. Some common arguments are:
   ```
 * To evaluate a model's performance 8 NPUs/GPUs:
   ```
-  mpirun --allow-run-as-root -n 8 python test.py --config ./configs/yolov7/yolov7.yaml --weight /path_to_ckpt/WEIGHT.ckpt --is_parallel True
+  msrun --worker_num=8 --local_worker_num=8 --bind_core=True --log_dir=./yolov7_log python test.py --config ./configs/yolov7/yolov7.yaml --weight /path_to_ckpt/WEIGHT.ckpt --is_parallel True
```

Comment:
这里也加一下
 Notice that if you are using msrun startup with 2 devices, please add `--bind_core=True` to avoid known performance error. For example:
```shell
    msrun --bind_core=True --worker_num=2--local_worker_num=2 --master_port=8118 \
    --log_dir=msrun_log --join=True --cluster_time_out=300 \
    python train.py --distribute --model=densenet121 --dataset=imagenet --data_dir=/path/to/imagenet
    ```

   > For more information, please refer to https://www.mindspore.cn/tutorials/experts/en/r2.3.1/parallel/startup_method.html

给用户提供一个msrun的链接指南